jsx/src/components/ServerDashboard/ServerDashboard.test.js

```diff
@@ -350,25 +349,25 @@ test("Shows server details with button click", async () => {
 
   await act(async () => {
     fireEvent.click(button);
+    jest.runAllTimers();
   });
-  clock.tick(400);
```

Comment:
clock.tick outside `act` caused problems. `runAllTimers` is a more robust, built in solution to the same problem.